DATA/EQUIPMENT/weapon_equip.ini

```diff
@@ -45208,8 +45208,8 @@ ids_name = 459606
 ; HYPERION Launcher ($class)
 ids_info = 461522
 ;res html
-; \m\Z00-01 “HYPERION” Launcher ($class)
-; \

```

Comment:
Why is there class info in the title of the infocard? AFAIK we never do that. Shouldn't that be in the description below?

Reply:
Yes, it's another error with this item that I missed in ini editing.